website/components/AppWrapper.tsx

```diff
@@ -76,7 +83,7 @@ export default function AppWrapper(): ReactElement {
 
   return (
     <div>
-      <ViewerStateProvider viewerSettings={viewerSettings}>
```

Comment:
Why did this get removed? (Was it because ShareModal still needed viewer state to be in ViewerStateProvider?)